addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -431,6 +431,91 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     );
 });
 
+QUnit.test('message list for tracking field', async function (assert) {
+    assert.expect(3);
+    await this.start();
+    this.data["mail.message"].fields.tracking_value_ids = {
+        relation: "mail.tracking.values",
+        string: "Tracking Values",
+        type: "many2many",
+    }
+    const thread = this.env.models['mail.thread'].create({
+        channel_type: 'channel',
+        id: 100,
+        members: [['insert', [
+            {
+                email: "john@example.com",
+                id: 9,
+                name: "John",
+            },
+        ]]],
+        model: 'mail.channel',
+        name: "General",
+        public: 'public',
+    });
+
+    this.data['mail.message'].records.push({
+        channel_ids: [100],
+        model: 'mail.channel',
+        res_id: 100,
+        id: 1,
+        tracking_value_ids: [{
+            changed_field: "Field 1",
+            field_type: "float",
+            id: 6,
+            new_value: 0,
+            old_value: 1,
+        }],
```

Comment:
Create thread and message in `this.data` before start (there are other examples in the code). Other tests in this file have to be adapted in another PR.

The goal is that if they are defined before start, they will be received at init messaging. Then the thread model can be found with `findFromIdentifyingData` instead of `create`